### ISSUE TITLE
Support for E-521

### ISSUE THREAD

**Issue by Cromefire** [openssl/openssl#5957]
As E-521 seems to be the curve with the biggest keysize of the [safecurves](https://safecurves.cr.yp.to/) it would be nice to have this included in openssl, to have a curve with large keysize, that's not a NIST one (which are possibly not safe according to safecurves)

**Comment by kroeckx** [comment]
Ed448-Goldilocks is the one that got standardized from that list, and which is now supported. You would need to publish a standard before we can use it.

**Comment by Cromefire** [comment]
As safecurves has parameters, I assume that there are parameters. Safecurves cites that: https://eprint.iacr.org/2013/647

**Comment by kroeckx** [comment]
I think you misunderstand my comment. If you want to use this for a key exchange, we need at least an assigned number to indicate it's that curve, but it's really more involved than that. If you want to use it in a certificate, we need a format and oid how to do this. It all seems unlikely to happen since E-521 was one of the candidates when the CFRG decided to recommend curve 25519 and curve 448. I suggest you go for curve 448.

**Comment by Cromefire** [comment]
Well, I'm not an expert on this topic so I think you will know more than I
